### PR TITLE
email settings banner

### PR DESCRIPTION
**Pull request by misspran**
## Description
Banner for letting users know that they can easily change their email subscriptions
## Screenshots / Demo Video
Screenshot 2023-03-29 at 2 40 08 PM

## Testing
manual testing steps:
- banner should show up when running app
- dismissed x button on banner should close the app
- settings should link to email preferences page to change them
- toggle should change email preferences for both grants and arpa tool
### Automated and Unit Tests
- [ ] Added Unit tests

### Manual tests for Reviewer
- [ ] Added steps to test feature/functionality manually

## Checklist
- [x] Provided ticket and description
- [x] Provided screenshots/demo
- [x] Provided testing information
- [ ] Provided adequate test coverage for all new code
- [ ] Added PR reviewers

### CODE REVIEW

File: packages/client/src/components/EmailSettingsBanner.vue [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import { mapActions, mapGetters } from 'vuex';
+
+export default {
+  props: {
+    showBanner: Boolean,
+    showProfileSettings: Function,
+  },
+  data() {
+    return {
+      checked: true,
```
Comment (as1729) [inline packages/client/src/components/EmailSettingsBanner.vue:33]: Does `checked: true` indicate to the user that they are already subscribed to all notifications? if so, we may want to flip it so the user has an option to subscribe to all notifications.

Comment (mhuang-usdr) [reply packages/client/src/components/EmailSettingsBanner.vue:33]: right now, by default the user is not subscribed to any notification. So the banner should show up as `checked: false` initially

Comment (misspran) [reply packages/client/src/components/EmailSettingsBanner.vue:33]: Ah sorry, ok will update

**Comment by misspran** [comment]
I just noticed in the settings visually in user preferences UI there’s 2 options:
`Grant assignments and New Grants Digest`
 but when you call the fetch the user there’s 3 email options a user can subscribe to.
```
GRANT_ASSIGNMENT
GRANT_DIGEST
GRANT_INTEREST
```
Is Grant Interest still a thing? Should I add Grant Interest toggle to the preferences UI? Sorry I asked this in like 3 different places.

**Review by as1729 — APPROVED** [review]
:shipit:

**Pull request merged**